src/Tempest/Core/src/functions.php

```diff
@@ -76,6 +76,6 @@ function env(string $key, mixed $default = null): mixed
      */
     function defer(Closure $closure): void
     {
-        get(DeferredTasks::class)->add($closure);
+    get(DeferredTasks::class)->add($closure);
```

Comment:
What's up with all these code style changes?

Reply:
Correct, I did not touch it deliberately.